ga/23.0.0.11/full/Dockerfile.ubuntu.ibmjava8

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM websphere-liberty:23.0.0.10-kernel-java8-ibmjava
+FROM websphere-liberty:kernel-java8-ibmjava
```

Comment:
Version should be included here and in other ubuntu-based Dockerfiles for full images.